src/process.py

```diff
@@ -30,6 +30,9 @@ def __init__(self, cachedir):
         self.good_firstname_count = 0
         self.bad_firstname_count = 0
         self.unknown_firstnames = Counter()
+        self.firstnames_expansion_map = {
+          "alb.": "Albert"
```

Comment:
Isn't firstname abbreviations expansion a can of worms? At least Albrecht, Albertine, Alban, Alberte and Albin might think so ;)

I'd rather not introduce false precision here and would prefer to offer possible variants in the frontend only, e.g. in a tooltip when hovering over the abbreviated name (ideally querying a database of (abbreviated) German first names of the first half of the 20th century, but such a thing might not exist).

Reply:
duh, you're absolutely correct, I should have added Alb. to givennames.csv, as an accepted abbreviation. Will amend and ping this pr when done.

Reply:
updated